packages/amplify-cli/src/index.ts

```diff
@@ -117,7 +100,7 @@ const normalizeStatusCommandOptions = (input: Input): Input => {
 /**
  * Command line entry point
  */
-export const run = async (startTime: number): Promise<number | undefined> => {
+export const run = async (startTime: number): Promise<number> => {
   try {
```

Comment:
Since you've already added handleError as an event listener for uncaughtException, I think this whole try / catch can be removed. Any error that gets bubbled all the way up will be handled by handleError automatically

Reply:
updated